packages/neon/neon_dashboard/lib/src/widgets/widget_item.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:neon/blocs.dart';
+import 'package:neon/theme.dart';
+import 'package:neon/utils.dart';
+import 'package:neon/widgets.dart';
+import 'package:nextcloud/dashboard.dart' as dashboard;
+
+class DashboardWidgetItem extends StatelessWidget {
+  const DashboardWidgetItem({
+    required this.item,
+    required this.roundIcon,
+    super.key,
+  });
+
+  final dashboard.WidgetItem item;
+  final bool roundIcon;
+
+  @override
+  Widget build(final BuildContext context) {
+    Widget leading = SizedBox.square(
+      dimension: largeIconSize,
+      child: NeonImageWrapper(
+        borderRadius: roundIcon ? BorderRadius.circular(largeIconSize) : null,
+        child: NeonCachedImage.url(
+          url: item.iconUrl,
+          account: NeonProvider.of<AccountsBloc>(context).activeAccount.value,
+          size: const Size.square(largeIconSize),
+        ),
+      ),
+    );
+    if (item.overlayIconUrl.isNotEmpty) {
+      leading = Stack(
+        children: [
+          leading,
+          SizedBox.square(
+            dimension: largeIconSize,
+            child: Align(
+              alignment: Alignment.bottomRight,
+              child: SizedBox.square(
+                dimension: smallIconSize,
+                child: NeonCachedImage.url(
+                  url: item.overlayIconUrl,
+                  account: NeonProvider.of<AccountsBloc>(context).activeAccount.value,
```

Comment:
Usage of the AccountsBloc is discouraged.

Do you think we can maybe extend `NeonApiImage` to fit here?

Reply:
Didn't you refactor this so passing the active account isn't necessary anymore? I wanted to use that, but couldn't find it :thinking:

Reply:
We have the mentioned NeinApiImage but that can only use api endpoints (DynamiteRawResponses).

But maybe we can sorta extend it.

Reply:
Solved in https://github.com/nextcloud/neon/pull/969